model_compiler/src/model_compiler/compilers/__init__.py

```diff
@@ -15,9 +15,10 @@
     openvino_model_file_to_openvino_model
 
 try:
```

Comment:
You should split these try cases in to two `try ... except` code piece, think about this case: `onnx_model_to_tensorrt_model` causes an error, but `onnx_model_file_to_enflame_model` is ok.